etcdctl/ctlv3/command/printer.go

```diff
@@ -174,19 +174,29 @@ func makeMemberListTable(r v3.MemberListResponse) (hdr []string, rows [][]string
 }
 
 func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]string) {
-	hdr = []string{"endpoint", "ID", "version", "db size", "is leader", "raft term", "raft index", "raft applied index", "errors"}
+	hdr = []string{"endpoint", "health", "error", "ID", "version", "db size", "is leader", "raft term", "raft index", "raft applied index", "resp errors"}
 	for _, status := range statusList {
-		rows = append(rows, []string{
-			status.Ep,
-			fmt.Sprintf("%x", status.Resp.Header.MemberId),
-			status.Resp.Version,
-			humanize.Bytes(uint64(status.Resp.DbSize)),
-			fmt.Sprint(status.Resp.Leader == status.Resp.Header.MemberId),
-			fmt.Sprint(status.Resp.RaftTerm),
-			fmt.Sprint(status.Resp.RaftIndex),
-			fmt.Sprint(status.Resp.RaftAppliedIndex),
-			fmt.Sprint(strings.Join(status.Resp.Errors, ", ")),
-		})
+		if status.Resp == nil {
+			rows = append(rows, []string{
+				status.Ep,
+				fmt.Sprint(status.Health),
+				fmt.Sprint(status.Error),
+			})
```

Comment:
@rleungx, thanks for the contribution! Just a little knit but as these first 3 rows are on each side of your if/else statement. Could you rewrite this so that you only have a single `if status.Resp != nil`?